images/10-virtualboxtools/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM gcc:7.3.0 as build-essential
+ENV KERNEL_VERSION 4.14.85-rancher
+
+ENV KERNEL_HEADERS https://github.com/rancher/os-kernel/releases/download/v${KERNEL_VERSION}/build-linux-${KERNEL_VERSION}-x86.tar.gz
+ENV KERNEL_BASE https://github.com/rancher/os-kernel/releases/download/v${KERNEL_VERSION}/linux-${KERNEL_VERSION}-x86.tar.gz
+ENV VBOX_VERSION 5.2.22
+ENV VBOX_SHA256 e51e33500a265b5c2d7bb2d03d32208df880523dfcb1e2dde2c78a0e0daa0603
+
+RUN apt-get update; \
+	apt-get install -y --no-install-recommends p7zip-full libelf-dev; \
+	apt-get clean; \
+	rm -rf /var/lib/apt/*
+RUN mkdir -p /usr/src/v${KERNEL_VERSION}; \
+	curl -sfL ${KERNEL_BASE} | tar zxf - -C /; \
+	curl -sfL ${KERNEL_HEADERS} | tar zxf - -C /usr/src/v${KERNEL_VERSION}
+RUN wget -O /vbox.iso "https://download.virtualbox.org/virtualbox/$VBOX_VERSION/VBoxGuestAdditions_$VBOX_VERSION.iso"; \
```

Comment:
better move to prebuild.sh

Reply:
No need to move to prebuild.sh because I don't need to install the `wget` package